bazel/grpc_build_system.bzl

```diff
@@ -112,10 +112,7 @@ def grpc_cc_library(
         visibility = visibility,
         testonly = testonly,
         linkopts = linkopts,
-        includes = [
-            "include",
-            "src/core/ext/upb-generated",
-        ],
+        includes = ["include"] + if_not_windows(["src/core/ext/upb-generated"]),
```

Comment:
nit: add a TODO?

Reply:
UPB is heavily reliant on POSIX and Linux headers, so it's not compatible with Windows in the near future.